Bottom: 17a47d03c923547e2a42ba9ec06fc6fff21277b7
Top:    2a9fbc811cb773be708676f29dff51bcd5e21ead
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-07 15:18:06 +0800

Refresh of macvlan-count-the-number-of

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index fed7dda..b7c814d 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -1481,6 +1481,7 @@ static int macvlan_changelink(struct net_device *dev,
 			      struct netlink_ext_ack *extack)
 {
 	struct macvlan_dev *vlan = netdev_priv(dev);
+	struct macvlan_port *port = vlan->port;
 	enum macvlan_mode mode;
 	bool set_mode = false;
 	enum macvlan_macaddr_mode macmode;
